src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.sections {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
It seems that the `.sections` class is not relevant to the `calendar.scss` file. It would be better to move this to a more general file, like `main.scss` or `index.scss`, since it seems to be a utility class.